SU2_CFD/src/solvers/CSolverFactory.cpp

```diff
@@ -429,7 +429,7 @@ CSolver* CSolverFactory::CreateFlowSolver(SUB_SOLVER_TYPE kindFlowSolver, CSolve
       break;
     case SUB_SOLVER_TYPE::NEMO_EULER:
       flowSolver = new CNEMOEulerSolver(geometry, config, iMGLevel);
-      flowSolver->Preprocessing(geometry, solver, config, iMGLevel, NO_RK_ITER, RUNTIME_FLOW_SYS, false);
+      //flowSolver->Preprocessing(geometry, solver, config, iMGLevel, NO_RK_ITER, RUNTIME_FLOW_SYS, false);
```

Comment:
Don't leave commented code please.

Reply:
I left as a comment first, because I am still not sure if I should delete this line of code or not. Since the CEulerSolver uses it.